libraries/Microsoft.Bot.StreamingExtensions/Payloads/Assemblers/ContentStreamAssembler.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Bot.StreamingExtensions.Payloads
+{
+    internal class ContentStreamAssembler : PayloadAssembler
+    {
+        private readonly IStreamManager _streamManager;
+
+        public ContentStreamAssembler(IStreamManager streamManager, Guid id)
+            : base(id)
+        {
+            ContentType = null;
+            ContentLength = null;
```

Comment:
remove this. Init, in the property, to "string.empty" and avoid the null programming craziness.